adam-core/src/test/scala/org/bdgenomics/adam/util/ADAMFunSuite.scala

```diff
@@ -25,6 +25,7 @@ trait ADAMFunSuite extends SparkFunSuite {
   override val properties: Map[String, String] = Map(("spark.serializer", "org.apache.spark.serializer.KryoSerializer"),
     ("spark.kryo.registrator", "org.bdgenomics.adam.serialization.ADAMKryoRegistrator"),
     ("spark.kryoserializer.buffer.mb", "4"),
-    ("spark.kryo.referenceTracking", "true"))
+    ("spark.kryo.referenceTracking", "true"),
+    ("spark.hadoop.parquet.task.side.metadata", "false"))
```

Comment:
In `ParquetInputFormat.java`, there is the following comment,

```java
/**
   * key to turn on or off task side metadata loading (default true)
   * if true then metadata is read on the task side and some tasks may finish immediately.
   * if false metadata is read on the client which is slower if there is a lot of metadata but tasks will only be spawn if there is work to do.
   */
  public static final String TASK_SIDE_METADATA = "parquet.task.side.metadata";
```

which I read to mean that setting this to `false` would mean that metadata is read on the client side instead of the task side. I thought that "task side" is the opposite of "client side" but I could very well be misunderstanding this too.